Core/Types/GameComparatorGRAS.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace CKAN
+{
+    /// <summary>
+    /// Test to see if the user's game is "Generally Recognised As Safe" (GRAS) with a given mod,
+    /// with extra understanding of which KSP versions are "safe" (ie: 1.0.5 mostly works with 1.0.4 mods).
+    /// If the mod has `ksp_version_strict` set then this is identical to strict checking.
+    /// </summary>
+    public class GameComparatorGRAS : IGameComparator
```

Comment:
So [acronyms longer than two characters are `PascalCased`](http://stackoverflow.com/questions/2109171), so `GameComparatorGras`. (This also means the top level namespace really ought to be `Ckan`...) Also, descriptors and adjective modifiers typically occur before the noun in a class name, following regular english conventions. So `GrasGameComparator`. Unfortunately this means similar classes are not sorted next to each other in an alphabetical list, but a common namespace could do instead.